app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,14 @@
+class MerchantsController < ApplicationController
+
+  def index
+    response = Faraday.get("http://localhost:3000/api/v1/merchants")
+    @merchants = JSON.parse(response.body, symbolize_names: true)[:data]
```

Comment:
It looks like you've got a refactor started with the service and the poro. Once you have a facade in place, you should be able to remove the calls to the API out of the controller completely. Same for the `show` action.